data/mods/TEST_DATA/known_good_uncrafts.json

```diff
@@ -39,6 +39,7 @@
       "art_trident",
       "art_urchin",
       "artifact_scp_doc",
+      "boomer_bile",
```

Comment:
@GuardianDll ok how is this not fixing the failed test?

Reply:
It did, but it fails with boomer_bile_adv now
Also it needs to be in a separate PR, not this one

Reply:
```suggestion
```